4-auth/structured_data/active_record/app/controllers/user_books_controller.rb

```diff
@@ -11,6 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START user_books]
```

Comment:
@remi it's because of the copying of these classes from the `structured_data` directory into the main app. This code had previously diverged, so I am adding those changes (the doc tags) back into the copied code for consistency.